frontend/src/components/SearchInputGroup.vue

```diff
@@ -9,14 +9,22 @@
           </div>
 
           <ComboboxInput v-if="selectedItem == null" v-focus class="w-full h-10 rounded-l-md border border-gray-300 bg-white py-2 px-10 shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary sm:text-sm disabled:bg-primary-l2" placeholder="John Doe" @change="query = $event.target.value"/>
-          <input v-else v-model="selectedItem.name" class="w-full h-10 rounded-l-md border border-gray-300 bg-primary-l2 py-2 px-10 shadow-xs focus:border-primary focus:outline-hidden focus:ring-1 focus:ring-primary sm:text-sm" readonly />
+          <div v-else class="w-full h-10 rounded-l-md border border-gray-300 bg-primary-l2 py-2 px-10 flex items-center justify-between shadow-xs sm:text-sm">
+            <span class="truncate">{{ selectedItem.name }}</span>
+            <span v-if="selectedItem.type === 'GROUP'" class="text-gray-500 text-xs italic mr-6">
+              {{ selectedItem.memberSize }} {{ t('vaultDetails.sharedWith.members') }}
+            </span>
+          </div>
         </div>
 
         <ComboboxOptions v-if="selectedItem == null && filteredItems.length > 0" class="absolute z-10 mt-1 max-h-56 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black/5 focus:outline-hidden sm:text-sm">
           <ComboboxOption v-for="item in filteredItems" :key="item.id" :value="item" class="relative cursor-default select-none py-2 pl-3 pr-9 ui-not-active:text-gray-900 ui-active:bg-primary ui-active:text-white">
             <div class="flex items-center">
               <img :src="item.pictureUrl ?? ''" alt="" class="h-6 w-6 shrink-0 rounded-full" >
               <span class="ml-3 truncate">{{ item.name }}</span>
+              <span v-if="item.type === 'GROUP'" class="ml-auto text-xs text-gray-500 italic">
+                {{ item.memberSize }} {{ t('vaultDetails.sharedWith.members') }}
```

Comment:
This is not `vaultDetails` and also use the placeholder approach (`{0} Members`)